### ISSUE TITLE
Readme breakage

### ISSUE THREAD

**Issue by kpixley** [rs/zerolog#224]
From the readme:  "zerolog.SetGlobalLevel: Can raise the minimum level of all loggers. Set this to zerolog.Disabled to disable logging altogether (quiet mode)."

This is incorrect.  SetGlobalLevel is not a variable.  it's a function.  So the line should be: "zerolog.SetGlobalLevel: Can raise the minimum level of all loggers. Call this with zerolog.Disabled to disable logging altogether (quiet mode).  zerolog.SetGLobalLevel(zerolog.Disabled)".

**Comment by rs** [comment]
Can you please submit a PR?

**Comment by kpixley** [comment]
Er... sorry.  What did I just do if not a PR?

**Comment by rs** [comment]
You opened an issue.

**Comment by kpixley** [comment]
Oh!  You mean a pull request.  I'm so used to PR standing for "problem report" I didn't understand.

No, I'm sorry.  I don't currently have the authority to give away the IP of my current employer.  I can point you to the problem, even recommend a fix, but I can't write it for you.  Doing so would require involving a couple of dozen other people, lawyers, meetings, executive sign-offs, etc.  That might be worthwhile if I were hoping to join the development team and could show value add for the company but I'm not.  Not at this point, anyway.

I'd have done that originally if I thought it would be efficient or effective for either of us.

**Comment by rs** [comment]
How can fix of a bug in an open source project not even published by your employer could be part of their IP?

**Comment by kpixley** [comment]
Because I wrote it.  Anything I write on their time clock belongs to them initially.

They can chose to give it back to you, if they want to.  Or to open source my changes separately from the original project.  Or any number of other things.  That's what all the meetings would be about.

Ethically, sure, you and I can agree that it would be easier if I wrote the change rather than reporting it and whether I/we retain the copyright and open source it our transfer the copyright to the project doesn't really matter so long as the code stream stays open source.  (Different projects have different policies on this, btw.)  But in practice, my employer owns the copyright of anything I write for them and even thinking about or discussing a possible grant costs time and money this probably isn't worth.  You and I have already spent more time discussing it than it would have cost someone else to simply have made the change.

I could probably have written it on my own time.  That would have been arguable, (since it would have been related to my work), but defensible since it was not on their time, so I'd likely have gotten away with it.  It's too late for that now, though, as I've already reported it under an account with their email address.  So I, personally, am burned on this issue.
